platforms/nuttx/src/bootloader/stm/stm32_common/main.c

```diff
@@ -71,6 +71,14 @@ const mcu_rev_t silicon_revs[] = {
 	{MCU_REV_Z, 'Z'}, /* Revision Z */
 };
 
+/*
+ * If APP_RESERVATION_SIZE is greater than 0 and
+ * FLASH_BASED_PARAMS is defined, throw a compile error
```

Comment:
The comment says the opposite of what it does below instead of explaining why. Could you please clarify? 😇

Reply:
Probably should say is not greater.